Makefile

```diff
@@ -6,3 +6,6 @@ fmt:
 
 test:
 	@go test -cover -v ./...
+
+docker:
+	docker build --rm -t zabio3/godolint .
```

Comment:
I assume https://hub.docker.com/u/zabio3 is you

Reply:
Yes, this is my account so I would like to push it after merge.